File: solutions/csharp/two-fer/1/TwoFer.cs
public static class TwoFer
{
    // In order to get the tests running, first you need to make sure the Speak method 
    // can be called both without any arguments and also by passing one string argument.
    public static string Speak(string name = "you")
    {
        string sentence = "One for {0}, one for me.";
        return String.Format(sentence, name);
    }
}

File: solutions/cpp/two-fer/1/two_fer.cpp
#include "two_fer.h"

namespace two_fer {

// TODO: add your solution here
std::string two_fer(std::string name) {
    const std::string start = "One for ";
    const std::string end = ", one for me.";
    if(name == "") {
        return start + "you" + end;
    }
    return start + name + end;
}

}  // namespace two_fer
